models/news-models.js

```diff
@@ -45,8 +46,43 @@ const fetchArticlesById = (article_id) => {
     })
 }
 
+const checkCommentExists = (article_id) => {
+
+    let queryString = `
+    SELECT article_id FROM articles
+    WHERE articles.article_id=$1
+    `;
+
+    return db.query(queryString, [article_id]).then((response) => {
+        if (response.rowCount === 0) {
+            return Promise.reject({status: 404, msg: 'article_id does not exist'});
+        } else {
+            return response.rows;
+        }
+    })
+}
+
+const fetchCommentsById = (article_id) => {
+
+    let queryString = `
+    SELECT * FROM comments
+    WHERE comments.article_id=$1
+    ORDER BY created_at DESC
+    `;
+
+    return db.query(queryString, [article_id]).then((response) => {
+        if (response.rowCount === 0) {
```

Comment:
Lack of comments here is not a 404 as the client is not looking for anything specific. In the event that there are no comments on an article, then its not that the resource is 'not found' its that there are simply no comments. So its perfectly appropriate for you to respond with an empty array. 
Think of a instagram post: If there are no comments on it theres no need for a 'not found' below the image. Simply having no comments is enough for the user/client to know that theres no comments.